clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/api/ProcessesService.java

```diff
@@ -16,16 +16,18 @@
 
 package com.netflix.spinnaker.clouddriver.cloudfoundry.client.api;
 
+import com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v3.*;
```

Comment:
I think we prefer to import just the necessary classes rather than wildcard

Reply:
Ya I think my IntelliJ converts them when there are more than 5 from that package. Either way, made the change. Do you know if there is an agreed upon limit?